package.json

```diff
@@ -55,7 +55,7 @@
     "errorhandler": "^1.4.3",
     "express": "^4.13.4",
     "express-bunyan-logger": "^1.2.0",
-    "freshy": "https://github.com/groupby/freshy",
+    "freshy": "1.0.4",
     "http-status": "^1.0.1",
```

Comment:
The version of freshy on GitHub is 1.0.3, not 1.0.4, but the interface should still be compatible, since it is a patch